### PR TITLE
docs: Review the Template Operator README file

### LINKED ISSUE #118: Review the README
As @kyma-project/technical-writers were not added as CODEOWNERS the [README.md](https://github.com/kyma-project/template-operator/blob/main/README.md) was merged without language review. Let's review the README and make necessary structure changes if needed.

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- Review the Template Operator README file

**Related issue(s)**
Fixes #118

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,179 +1,192 @@
...
-  This guide is HIGHLY RECOMMENDED to be followed for a smooth development process.
-  This is a good alternative if you do not want to use an entire control-plane infrastructure and still want to properly test your operators.__
+* A provisioned Kubernetes cluster and OCI registry
 * [kubectl](https://kubernetes.io/docs/tasks/tools/)
 * [kubebuilder](https://book.kubebuilder.io/)
+
+Use one of the following options to install kubebuilder:
+
+<!-- tabs:start -->
+#### **Brew**
```
Comment (ruanxin) [inline README.md:127]:
```suggestion
#### **Homebrew**
```

File: README.md [hunk 2]
```diff
@@ -273,194 +285,182 @@ _WARNING: This step requires the working OCI Registry from our [Pre-requisites](
...
 If required, you can adjust this Kyma CR based on your testing scenario. For example, if you are running a dual-cluster setup, you might want to enable the synchronization of the Kyma CR into the runtime cluster for a full E2E setup.
-On creation of this kyma CR in your Control Plane cluster, installation of the specified modules should start immediately.
+When creating this Kyma CR in your Control Plane cluster, installation of the specified modules should start immediately.
 
-### Debugging the operator ecosystem
+### Debugging the Operator Ecosystem
 
 The operator ecosystem around Kyma is complex, and it might become troublesome to debug issues in case your module is not installed correctly.
-For this very reason here are some best practices on how to debug modules developed through this guide.
+For this reason here are some best practices on how to debug modules developed through this guide.
```
Comment (nataliasitko) [inline README.md:463]:
```suggestion
For this reason, here are some best practices on how to debug modules developed using this guide.
```

File: README.md [hunk 3]
```diff
@@ -472,16 +472,15 @@ For this very reason here are some best practices on how to debug modules develo
     JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}' \
     && kubectl get manifest -o jsonpath="$JSONPATH"-n kcp-system
    ```
-3. Depending on your issue, either observe the deployment logs from either `lifecycle-manager` and/or `module-manager`. Make sure that no errors have occurred.
+3. Depending on your issue, either observe the deployment logs from either Lifecycle Manager or Module Manager. Make sure that no errors have occurred.
```
Comment (nataliasitko) [inline README.md:475]:
```suggestion
3. Depending on your issue, observe the deployment logs from either Lifecycle Manager or Module Manager. Make sure that no errors have occurred.
```

File: README.md [hunk 4]
```diff
@@ -472,16 +472,15 @@ For this very reason here are some best practices on how to debug modules develo
...
 
-Usually the issue is related to either RBAC configuration (for troubleshooting minimum privileges for the controllers, see our dedicated [RBAC](#rbac) section), mis-configured image, module registry or `ModuleTemplate`.
-As a last resort, make sure that you are aware if you are running within a single-cluster or a dual-cluster setup, watch out for any steps with a `WARNING` specified and retry with a freshly provisioned cluster.
-For cluster provisioning, please make sure to follow our recommendations for clusters mentioned in our [Pre-requisites](#pre-requisites) for this guide.
+Usually, the issue is related to either RBAC configuration (for troubleshooting minimum privileges for the controllers, see our dedicated [RBAC](#role-based-access-control-rbac) section), misconfigured image, module registry or `ModuleTemplate`.
+As a last resort, make sure that you are running within a single-cluster or a dual-cluster setup, watch out for any steps with a `WARNING` specified and retry with a freshly provisioned cluster.
+For cluster provisioning, make sure to follow our recommendations for clusters mentioned in our [Prerequisites](#prerequisites) for this guide.
```
Comment (nataliasitko) [inline README.md:479]:
```suggestion
For cluster provisioning, make sure to follow the recommendations for clusters mentioned in our [Prerequisites](#prerequisites) for this guide.
```

File: README.md [hunk 5]
```diff
@@ -472,16 +472,15 @@ For this very reason here are some best practices on how to debug modules develo
...
 
-Lastly, if you are still unsure, please feel free to open an issue, with a description and steps to reproduce, and we will be happy to help you with a solution.
+Lastly, if you are still unsure, open an issue with a description and steps to reproduce. We will be happy to help you with a solution.
```
Comment (nataliasitko) [inline README.md:481]:
```suggestion
Lastly, if you are still unsure, [open an issue](https://github.com/kyma-project/template-operator/issues/new/choose) with a description and steps to reproduce. We will be happy to help you with a solution.
```

File: README.md [hunk 6]
```diff
@@ -273,194 +285,182 @@ _WARNING: This step requires the working OCI Registry from our [Pre-requisites](
...
+When creating this Kyma CR in your Control Plane cluster, installation of the specified modules should start immediately.
 
-### Debugging the operator ecosystem
+### Debugging the Operator Ecosystem
 
 The operator ecosystem around Kyma is complex, and it might become troublesome to debug issues in case your module is not installed correctly.
-For this very reason here are some best practices on how to debug modules developed through this guide.
+For this reason here are some best practices on how to debug modules developed through this guide.
 
 1. Verify the Kyma installation state is ready by verifying all conditions.
```
Comment (nataliasitko) [inline README.md:465]:
```suggestion
1. Verify the Kyma installation state is `Ready` by verifying all conditions.
```
Please use code font for "Ready" also in point 2

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**